src/org/skypro/skyshop/basket/ProductBasket.java

```diff
@@ -3,12 +3,13 @@
 import org.skypro.skyshop.product.Product;
 import org.skypro.skyshop.product.SimpleProduct;
 
-import java.util.Random;
+import java.util.*;
 
 public class ProductBasket {
-    private static final int maxProducts = 5;
-    private final Product[] products = new Product[maxProducts];
-    private int count = 0; // счетчик продуктов в корзине
+    //private static final int maxProducts = 5;
+    //private final Product[] products = new Product[maxProducts];
+    LinkedList<Product> products = new LinkedList<>();
```

Comment:
Для объявления переменных лучше использовать абстракции
```suggestion
    List<Product> products = new LinkedList<>();
```